README.md

```diff
@@ -1,6 +1,6 @@
 # ShadowNode
 
-The Node.js Runtime in shadow, which supports NPM packages partially for embeddable platform.
+The Node.js runtime in shadow, enables N-API and vast Node.js packages on embedded devices.
```

Comment:
"The Node.js runtime in device"?

Reply:
The "Node.js packages" does include  "packages from NPM", but not vice versa since not all packages are published to NPM.